docs/openapi/components/schemas/credentials/USMCACertificationOfOrigin.yml

```diff
@@ -131,10 +456,17 @@ properties:
               title: Description
               description: Description of the shipment.
               type: string
+            gtin:
```

Comment:
hopefully this produces JSON-LD that recognizes GS1 controls this term

Reply:
It's defined as `schema.org/gtin` in https://github.com/w3c-ccg/traceability-vocab/blob/main/docs/openapi/components/schemas/common/Product.yml#L26